nightingale_utils/src/utils.py

```diff
@@ -56,8 +58,35 @@ def room_pose_lookup(self, req):
         return res
 
 
+class RobotConfigurationService:
+    def __init__(self):
+        self.ss = rospy.Service(
+            "/nightingale/robot_configuration_lookup",
+            RobotConfigurationLookup,
+            self.robot_configuration_lookup,
+        )
+        self.configurations = rospy.get_param("/nightingale_utils/robot_configurations")
+
+    def robot_configuration_lookup(self, req):
```

Comment:
Nit: Add a check to make sure all the lookup arguments are defined in self.configurations